examples/example3/local_adapter.py

```diff
@@ -0,0 +1,55 @@
+import json
+
+from slugify import slugify
+
+from diffsync import DiffSync
+from models import Region, Country
+
+COUNTRIES_FILE = "countries.json"
+
+
+class LocalAdapter(DiffSync):
+    """DiffSync Adapter to Load the list of regions and countries from a local JSON file."""
+
+    region = Region
+    country = Country
+
+    # Since all countries are associated with a region, we don't need to list country here
+    # When doing a diff or a sync between 2 adapters,
+    #  diffsync will recursively check all models defined at the top level and their children.
```

Comment:
```suggestion
    # When doing a diff or a sync between 2 adapters,
    #  DiffSync will recursively check all models defined at the top level and their children.
    # Since our data model defines countries as children of a region, we don't need to list country here
```